lib/FreeRTOS/mqtt-agent-interface/freertos_command_pool.c

```diff
@@ -58,7 +58,7 @@ static Command_t commandStructurePool[ MQTT_COMMAND_CONTEXTS_POOL_SIZE ];
  * may be obtained by receiving a pointer from the queue, and returned by
  * sending the pointer back into it.
  */
-static AgentMessageContext_t commandStructQueue;
+static AgentMessageContext_t commandStructMessageCtx;
```

Comment:
The comment needs to be updated too.

Reply:
Since this is a FreeRTOS-only file, and we're using the FreeRTOS queue implementation defined in `freertos_agent_message.h`, I left the references to queues but added a comment about why this is a queue